src/Bicep.Cli.IntegrationTests/BuildCommandTests.cs

```diff
@@ -104,9 +104,7 @@ public async Task Build_Valid_SingleFile_ToStdOut_ShouldSucceed(DataSet dataSet)
 
             if (dataSet.HasExternalModules)
             {
-                // ensure something got restored
-                Directory.Exists(settings.Features.CacheRootDirectory).Should().BeTrue();
-                Directory.EnumerateFiles(settings.Features.CacheRootDirectory, "*.json", SearchOption.AllDirectories).Should().NotBeEmpty();
+                settings.Features.Should().HaveValidModules();
```

Comment:
The old code checks both br and ts modules, but it seems HaveValidModules only checks br modules, so we may need to update it to also check ts modules. I can make the change in my PR to add file locks for ts modules.